akri/README.md

```diff
@@ -0,0 +1,53 @@
+We recommend Akri to enable leaf device access on the cluster.
+
+## Prerequisites:
+Before performing the procedure in this topic, you must have installed and configured the following:
+
+* A Kubernetes cluster created with AKS-HCI with at least 1 master and 1 Linux worker nodes.
+* Install Helm V3 and add it to system PATH. 
+  Helm is the package manager for Kubernetes that runs on a local machine with `kubectl` access to the Kubernetes cluster. The installation process for Prometheus and the Certificate Manager leverage Helm charts available on the public Helm repo. Please review the steps on your own recommended way of [Installing Helm](https://helm.sh/docs/using_helm/#installing-helm).
+  Download and install the [Helm CLI](https://github.com/helm/helm/releases/tag/v3.3.0) on the local machine that will be interfacing with the Kubernetes cluster. 

```

Comment:
Should we not point to a specific release so as not to have an obsolete link soon?

Reply:
this is all copy from their other add-ons.  i agree, but want to be aligned with AKS-HCI